.env.development

```diff
@@ -23,7 +23,6 @@ AUTHN_MINIMAL_HEADER=true
 LOGIN_ISSUE_SUPPORT_LINK='/login-issue-support-url'
 TOS_AND_HONOR_CODE='http://localhost:18000/honor'
 PRIVACY_POLICY='http://localhost:18000/privacy'
-REGISTRATION_OPTIONAL_FIELDS=''
```

Comment:
Are we defining a new variable for optional fields on the progressive profiling page?

Reply:
none yet. We might not need it.